cmd/txt-suite/main.go

```diff
@@ -66,28 +66,29 @@ func run(tests []*test.Test) bool {
 		ioutil.WriteFile(logfile, data, 0664)
 	}
 
+	fmt.Printf("For more information about the documents and chapters, run: txt-suite -m\n\n")
 	for index := range tests {
 		if tests[index].Status == test.NotImplemented {
 			continue
 		}
 		if tests[index].Result == test.ResultNotRun {
 			continue
 		}
+		fmt.Printf("%02d - ", index)
 		fmt.Printf("%-40s: ", a.Bold(tests[index].Name))
 		f.Flush()
 
 		if tests[index].Result == test.ResultPass {
-			fmt.Printf("%-20s\n", a.Bold(a.Green(tests[index].Result)))
+			fmt.Printf("%-20s", a.Bold(a.Green(tests[index].Result)))
 		} else if tests[index].Result == test.ResultWarn {
-			fmt.Printf("%-20s\n", a.Bold(a.Yellow(tests[index].Result)))
+			fmt.Printf("%-20s", a.Bold(a.Yellow(tests[index].Result)))
 		} else {
-			fmt.Printf("%-20s\n", a.Bold(a.Red(tests[index].Result)))
+			fmt.Printf("%-20s", a.Bold(a.Red(tests[index].Result)))
 		}
 		if tests[index].ErrorText != "" {
-			fmt.Printf(" %s\n", tests[index].ErrorText)
-		}
-		if tests[index].ErrorTextSpec != "" {
```

Comment:
ErrorTextSpec has been removed, but no reason was given why.

Reply:
Output looks too complex. Document ids can be looked up via testplan

Reply:
too complex? Also the testplan isn't part of the binary.
How about adding a new flag (-v for verbose) and only print if if -v is specified.

Reply:
We already have a -m option which prints all necessary information.

Reply:
????????????